### ISSUE TITLE
Add support for connections througt a proxy

### ISSUE THREAD

**Issue by scarballo** [hotelbeds-sdk/hotel-api-sdk-java#55]
It would be nice to add proxy support with credentials in your client. This could be made in a easy way while initialization with a custom constructor or using a fluent pattern that covers your client "builder" adding the posibility of add new options.

Would be this possible?

**Comment by Verdoso** [comment]
It should be fairly easy to add, as HotelApiClient uses OkHttpClient internally and that library already supports proxy configuration.

It should be as simple as adding the appropiate configuration options and add the proxy, if necessary, in the HotelApiClient.init() method (Line 191 of [HotelApiClient.java](https://github.com/hotelbeds-sdk/hotel-api-sdk-java/blob/master/hotel-api-sdk/src/main/java/com/hotelbeds/distribution/hotel_api_sdk/HotelApiClient.java)). Something like that is shown at [this stackoverflow question](https://stackoverflow.com/questions/37866902/okhttp-proxy-settings).

Cheers,

**Comment by scarballo** [comment]
Yes, I know. I'ts a very easy and short time development. Should we expect this feature in your next release? It would be nice.

**Comment by Verdoso** [comment]
This is something to be answered by the HB developers, I'm just passerby ;D
